docs/technotes/document-status.rst

```diff
@@ -6,6 +6,7 @@ Technotes are continuously updated on the web as you add commits and merge pull
 Your "drafts" are visible to the rest of the observatory, and even the public, to speed up collaboration and provide transparency.
 You can communicate the status of the document by setting metadata in :file:`technote.toml`.
 For example, you can specify that the technote is an incomplete draft, or that the technote is deprecated and is now replaced by another set of documents.
+The field is optional: if for some reason none of these values describe the status of your technote, remove (or comment out) the technote.state code block.
```

Comment:
This isn't quite right. If none of the values describe the status of your document you can use the "other" state and then put a "note" in place. So this would be a third section.

Here's an alternative to this sentence describing the default:

```suggestion
This configuration is optional; the default status is "stable" when the ``[technote.state]`` table is omitted.
```